13/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="ua">
```

Comment:
There is a mix of languages on the page 

<img width="1602" alt="image" src="https://user-images.githubusercontent.com/84010501/200113057-bc202697-352f-4c3c-a28a-003a091d553c.png">